packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -389,6 +413,8 @@ class _RenderCheckbox extends RenderToggleable {
          hovering: hovering,
        );
 
+  bool useActiveColorInDisabledState;
```

Comment:
Good call, I added some dart docs explaining why we need this bool is needed.

I'd prefer not to add it to the RenderToggleable, since it is only needed for the implementation of Switches, Radio buttons, and checkboxes, and adding it to RenderToggleable would make it public API.